mlton/backend/backend.fun

```diff
@@ -461,9 +498,15 @@ fun toMachine (rssa: Rssa.Program.t) =
          in
             case s of
                Bind {dst = (var, _), src, ...} =>
-                  Vector.new1
-                  (M.Statement.move {dst = varOperand var,
-                                     src = translateOperand src})
+                  let
+                     val oper = varOperand var
+                  in
+                     if M.Operand.isDestination oper
```

Comment:
Since Statics come from statements in rssa, they occur in a Bind statement. Without this step, backend would output the invalid instruction to bind a static. So yes it's more or less necessary.

Reply:
But, RSSA can also `Bind` a `Const` operand to a variable.  So, why wasn't this check already here to avoid emitting an invalid instruction to move into a `Const`?  I guess because of RSSA copy propagation --- although the RSSA IR allows `Bind`ing a `Const`, it never occurred in the RSSA program that was processed by the backend.

Reply:
I believe that's the case. Consts propagate in Rssa but Statics don't until they're set up in the backend.